readme.md

```diff
@@ -12,13 +12,19 @@
 
 [Screencast on Youtube](https://www.youtube.com/watch?v=JxW8lJzLhxI)
 
-### References
+## Framing: Review One-to-Many Relationships (5 min)
 
-* [Rails Guides - Has Many Through](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association)
-* [Tunr Playlists Starter](https://github.com/ga-dc/tunr_rails/tree/playlists-starter)
-* [Tunr Playlists Solution](https://github.com/ga-dc/tunr_rails/tree/playlists-solution)
+So far in Class we have been working with One to Many Relationships:
+
+* In Scribble, we had `Posts` and `Comments`.
+* During our Hogwarts Lab, we had `Houses` and `Students`
+* In Tunr, we had `Artists` and `Songs`
 
-## Many-to-Many Relationships
+However, using the Tunr as an example, let's consider that a lot of music is composed collaboratively and we may have songs that belong to **multiple Artists**.
```

Comment:
since they are going to be doing playlists anyway maybe use that as the framing? `A song can belong to multiple playlists`